src/settings/settings.scss

```diff
@@ -44,24 +55,47 @@ body {
   text-align: center;
   font-size: $s-font-size-desktop;
   font-weight: $font-bold;
+
+  @media(max-width: $l-breakpoint) {
+    width: 350px;
+    height: 95px;
+  }
+
+  @media(max-width: $m-breakpoint) {
+    width: 225px;
+    height: 71px;
+    font-size: $s-font-size-mobile;
+  }
 }
 
 .section__button--save {
   background: transparent;
-  margin: 10px;
   border: 0;
+  width: 150px;
   font-size: $m-font-size-desktop;
-  border-bottom: 1px solid $white-color;
+  border-bottom: 3px solid $white-color;
   color: $white-color;
+
+  @media(max-width: $m-breakpoint) {
+    width: 101px;
+    font-size: $m-font-size-mobile;
+    margin-bottom: 25px;
+  }
 }
 
 .section__link--return {
-  font-size: $s-font-size-desktop;
+  font-size: $m-font-size-desktop;
   text-decoration: none;
   align-self: flex-end;
   margin-right: 79px;
+
+  @media(max-width: $m-breakpoint) {
+    font-size: $s-font-size-mobile;
+    margin-right: 0;
+    align-self: center;
+  }
 }
 
 .section__link--return:visited {
   color: $white-color;
-}
+}
```

Comment:
Like we discussed on Discord - there is a bit to change in styles like missing font-weights, colors, paddings/margins, cursor on the button, etc.